client/client.go

```diff
@@ -230,3 +232,23 @@ func (c *Client) SearchWord(word string) ([]string, error) {
 	sort.Strings(filenames)
 	return filenames, nil
 }
+
+// SearchWordStrict is similar to `SearchWord`, but it uses a `grep` command to
+// eliminate the possible false positives.
```

Comment:
Mention that the given keyword must match exactly the form of the keyword in the file.